packages/datepicker-react/package.json

```
@@ -40,14 +40,12 @@
         "@fremtind/jkl-react-hooks": "^1.6.11",
         "@fremtind/jkl-select-react": "^2.4.12",
         "@fremtind/jkl-text-input-react": "^3.5.8",
-        "@nrk/core-datepicker": "3.0.8",
         "classnames": "^2.2.6",
         "nanoid": "^3.1.10"
     },
     "devDependencies": {
-        "@fremtind/jkl-core": "^4.3.1",
         "@fremtind/jkl-portal-components": "^0.7.4",
-        "@mdx-js/parcel-plugin-mdx": "^1.5.7"
+        "@nrk/core-datepicker": "^3.0.8"
```

```suggestion
        "@nrk/core-datepicker": "3.0.8"
```

vi ville vel låse, som i accordion, da slipper du endringen i pkg.json også

---

Ja, kan godt låse det! Har nok ikke så mye å si, men digg å slippe diff i pkg/lockfila som du sier.